Description: Remove mentions of `sd_notify` library
 Calling shell helper systemd_notify(1) is enough to do the same.
 .
 Systemd unit file should specify `NotifyAccess=all`.
From: Alexey Lebedeff <alebedev@mirantis.com>
Date: Fri, 22 Jan 2016 17:26:00 +0300
Origin: upstream, https://github.com/binarin/rabbitmq-server/commit/466dea8ba68f5687eb8e2dfb90d06f94ce3ac3ee.patch
Last-Update: 2016-02-14

--- rabbitmq-server-3.6.0.orig/src/rabbit.erl
+++ rabbitmq-server-3.6.0/src/rabbit.erl
@@ -284,11 +284,10 @@ broker_start() ->
     Plugins = rabbit_plugins:setup(),
     ToBeLoaded = Plugins ++ ?APPS,
     start_apps(ToBeLoaded),
-    case code:load_file(sd_notify) of
-        {module, sd_notify} -> SDNotify = sd_notify,
-                               SDNotify:sd_notify(0, "READY=1");
-        {error, _} -> ok
-    end,
+    %% Only for systemd unit with Type=notify. Errors are intentionally
+    %% ignored: either you have working systemd-notify(1) or you don't
+    %% care about systemd at all.
+    os:cmd("systemd-notify --ready")
     ok = log_broker_started(rabbit_plugins:active()).
 
 start_it(StartFun) ->
